libs/s3/s3_multipart_test_lib.py

```diff
@@ -626,3 +628,40 @@ def complete_multipart_upload_with_di(
             raise CTException(err.S3_CLIENT_ERROR, error)
 
         return response
+
+    def create_mpu_get_precalc_parts(
```

Comment:
@archanabpatil this should included in a new library file - s3_multipart_common_lib in case assertions need to be retained, based on previous design discussions.

Alternatively this can be refactored to not include assertions as well and be retained in s3_multipart_test_lib itself.

Reply:
okay , will remove assertion here as its always expected to return true and the create_multipart_upload is handling the exception anyways